katran/lib/KatranLb.cpp

```diff
@@ -464,6 +464,7 @@ void KatranLb::featureDiscovering() {
   } else {
     features_.directHealthchecking = false;
   }
+  //TODO: implement discovery for localDeliveryOptimization feature
```

Comment:
what for? (this is also related to the part when this feature was added into enum). how do you see this knowledge would be usefull in userspace? what code could be broken if such gate wont exists?

Reply:
No part of the code would be broken. I've added this feature into the enum to be unified with existing features, where we have this discovery function for other features. IMO, this part is just for report existed features. 
Also, I've added this TODO because I haven't the idea how to do this (I've explained in this [comment](https://github.com/facebookincubator/katran/pull/107#issuecomment-743931839)).
Do you think that adding this feature to the enum is not necessary?

Reply:
i think "TODO" is not necessary, because most likely noone is ever will need it and/or work on it. but having feature in enum is good for housekeeping.